src/main/java/com/surrealdb/connection/SurrealWebSocketConnection.java

```diff
@@ -50,6 +50,20 @@ public SurrealWebSocketConnection(String host, int port, boolean useTls) {
         this.resultTypes = new HashMap<>();
     }
 
+    @SneakyThrows
+    public SurrealWebSocketConnection(String host, boolean useTls) {
+        super(URI.create((useTls ? "wss://" : "ws://") + host + "/rpc"));
```

Comment:
The host string should support URI format. Happy with default ports, but the implementation here is doing protocol prefixing - I would like to avoid that please